Set the FreeBSD /etc/salt dir to PREFIX/etc/salt

geoffgarside opened the pull request:
The FreeBSD port version of saltstack uses /usr/local/etc/salt rather
than /etc/salt for its configuration. This change fixes the use of a configuration directory when bootstrapping FreeBSD.

geoffgarside committed:
Set the FreeBSD /etc/salt dir to PREFIX/etc/salt

The FreeBSD port version of saltstack uses /usr/local/etc/salt rather
than /etc/salt for its configuration.

s0undt3ch commented on bootstrap-salt.sh:
```diff
@@ -1976,6 +1976,12 @@ install_freebsd_9_stable_deps() {
+
+    # Lets set SALT_ETC_DIR to ports default
+    if [ "$TEMP_CONFIG_DIR" != "null" ]; then
+        SALT_ETC_DIR="/usr/local/${SALT_ETC_DIR}"
```
Why not:

``` bash
SALT_ETC_DIR=${BS_SALT_ETC_DIR:-/usr/local/etc/salt}
```

geoffgarside replied:
Ah yes, as that will respect the environment variable as well won't it.

s0undt3ch replied:
Yes, it will.

geoffgarside committed:
Respect the environment.

Remove the check for the TEMP_CONFIG_DIR as the FreeBSD SALT_ETC_DIR is
irrespective of the TEMP_CONFIG_DIR.

s0undt3ch commented:
Thanks!
